packages/ui/src/pages/MyStudiesPage/components/RecentStudiesItem.vue

```diff
@@ -0,0 +1,73 @@
+<script setup lang="ts">
+import studiesAPI from "@/api/studies";
+import { useMutation } from "@tanstack/vue-query";
+
+const emit = defineEmits(["deleted"]);
+const props = defineProps<{ name: string; description: string; id: string }>();
+useMutation({
+  mutationFn: () => studiesAPI.deleteStudy(props.id),
+  onSuccess: () => {
+    emit("deleted");
+  },
+});
+</script>
+
+<template>
+  <RouterLink
+    :to="{ name: 'study', params: { id } }"
+    class="block hover:bg-neutral-50 transition-colors"
+  >
+    <div
```

Comment:
Can use Element Plus Card component (https://element-plus.org/en-US/component/card.html#card) instead of a div for each item. Also seems like there are a lot of nested divs here, can look into simplifying the HTML